util.js

```diff
@@ -66,6 +66,7 @@ function getDock() {
     const cosmicDock = Main.extensionManager.lookup("cosmic-dock@system76.com");
     if (cosmicDock
         && cosmicDock.stateObj
```

Comment:
Does the version of JS used in GNOME support `?.` operators?

If so, this could be shortened to `if (cosmicDock?.stateObj?.dockManager?.mainDock && cosmicDock.state === ExtensionUtils.ExtensionState.ENABLED)`

Reply:
Looks like `?.` appears in a bunch of places in Gnome Shell, so that should work.

(For reference, you can also test in `gjs`, which should be the save JavaScript interpreter used by Gnome Shell).

Reply:
yep, I just force pushed the change. I didn't realize that was also a JS thing, thanks!